DevEdu.API/DevEdu.DAL/Models/CommentDto.cs

```diff
@@ -1,11 +1,10 @@
-﻿using System;
-
-namespace DevEdu.DAL.Models
+﻿namespace DevEdu.DAL.Models
 {
     public class CommentDto : BaseDto
     {
         public string Text { get; set; }
         public UserDto User { get; set; }
+        public string Date { get; set; }
```

Comment:
it must be DateTime
strings are on the level of input/output-models
and DateTime - in DTO, because it's an representation of currently existing entity in DB